packages/arb-token-bridge-ui/src/components/common/SiteBanner.tsx

```diff
@@ -1,39 +1,92 @@
+import dayjs from 'dayjs'
+import { useEffect, useState } from 'react'
+import { twMerge } from 'tailwind-merge'
 import { ExternalLink } from './ExternalLink'
+import { ArbitrumStatusResponse } from '../../pages/api/status'
+import { getAPIBaseUrl } from '../../util'
 
-export const SiteBanner = ({
-  children,
-  ...props
-}: React.HTMLAttributes<HTMLDivElement>) => {
+const generateArbiscanIncidentMessage = () => {
   return (
-    <div
-      className="bg-atmosphere-blue px-4 py-[8px] text-center text-sm font-normal text-white"
-      {...props}
-    >
-      {children}
-    </div>
+    <p>
+      <ExternalLink className="underline" href="https://arbiscan.io/">
+        Arbiscan
+      </ExternalLink>{' '}
+      is temporarily facing some issues while showing the latest data. Arbitrum
+      chains are still live and running. If you need an alternative block
+      explorer, you can visit{' '}
+      <ExternalLink
+        className="underline"
+        href="https://www.oklink.com/arbitrum"
+      >
+        OKLink
+      </ExternalLink>
+      .
+    </p>
   )
 }
 
-export function SiteBannerArbiscanIncident() {
+export const SiteBanner = ({
+  children,
+  expiryDate,
+  ...props
+}: React.HTMLAttributes<HTMLDivElement> & { expiryDate?: string }) => {
+  const [arbitrumStatus, setArbitrumStatus] = useState<
+    ArbitrumStatusResponse | undefined
+  >()
+
+  useEffect(() => {
+    const fetchSetArbitrumStatus = async () => {
+      try {
+        const response = await fetch(`${getAPIBaseUrl()}/api/status`, {
+          method: 'GET',
+          headers: { 'Content-Type': 'application/json' }
+        })
+        const _arbitrumStatus: ArbitrumStatusResponse = (await response.json())
+          .data
+        setArbitrumStatus(_arbitrumStatus)
+      } catch (e) {
+        // error fetching status
+        console.error(e)
+      }
+    }
+    fetchSetArbitrumStatus()
+  }, [])
+
+  // show incident-banner if there is an active incident
+  const showArbiscanIncidentBanner =
+    (arbitrumStatus?.content.components || []).findIndex(
+      component =>
+        component.name.toLowerCase().includes('arbiscan') &&
+        component.status !== 'OPERATIONAL'
+    ) > -1
```

Comment:
with a couple of util functions we can make this a lot more readable (plus I don't think we should raise an incident for sepolia arbiscan being down

```
function isComponentArbiscan({ name }: { name: string }) {
  const componentNameLowercased = component.name.toLowerCase()
  return componentNameLowercased === 'arb1 - arbiscan' || componentNameLowercased === 'nova - arbiscan'
}

function isComponentOperational({ status }: { status: string }) {
  return status === 'OPERATIONAL'
}
```

we can use `Array.some`:

```suggestion

  const showArbiscanIncidentBanner =
    arbitrumStatus.content.components.some(
      component => isComponentArbiscan(component) && !isComponentOperational(component)
    )
```